skyportal/tests/conftest.py

```diff
@@ -8,15 +8,13 @@
 
 from baselayer.app import models
 from baselayer.app.config import load_config
-from baselayer.app.test_util import (
-    driver,
-    MyCustomWebDriver,
-    set_server_url,
-    reset_state,
-)
+from baselayer.app.test_util import driver  # noqa: F401
+from baselayer.app.test_util import MyCustomWebDriver  # noqa: F401
+from baselayer.app.test_util import set_server_url
+from baselayer.app.test_util import reset_state  # noqa: F401
```

Comment:
Had to do it like this, as `flake8` would otherwise ignore the `noqa`'s (tried putting them in everywhere..)

Reply:
Yes, errors get thrown if these are removed.